### PR TITLE
Adding Visual Editor and Flow ernaehrungsrathh

### PR DESCRIPTION
**Pull request by NeoMahler**
Requested at https://meta.miraheze.org/wiki/Request_features#ErnaehrungsratHH_.28Status:_Open.29

### CODE REVIEW

**Comment by revi** [comment]
VisualEditor needs puppet configurations before being merged.

**Comment by Reception123** [comment]
Done in miraheze/puppet/issues/118

**Comment by Reception123** [comment]
@revi @NeoMahler + it has to be ernaehrungsrathhwiki (all names have to end with wiki)

**Comment by NeoMahler** [comment]
@revi does Flow need puppet configurations?

**Comment by revi** [comment]
Yes, but it was done by reception123. However his comment is valid and you need to fix it.

**Comment by NeoMahler** [comment]
@revi Done

File: LocalSettings.php [hunk 1]
```diff
@@ -396,6 +396,7 @@
 		'spiralwiki' => true,
 		'universebuildwiki' => true,
 		'walthamstowlabourwiki' => true,
+		'ernaehrungsrathhwiki' => true
```
Comment (revi) [inline LocalSettings.php]: You need `,` everywhere, every time the line ends. Just FYI.

**Pull request merged**